CONTRIBUTING.md

```diff
@@ -48,21 +48,11 @@ $ bin/phpunit
 ```
 
 This command will only run tests which does not require a real web server. To run the full tests, you will need to
-copy/paste the `phpunit.xml.dist` to `phpunit.xml` and uncomment the `TEST_SERVER` php server variable. Then, you have
-three possibilities:
-
- * Rely on Apache2 (PHP) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/apache2-php.conf`.
- * Rely on Apache2 (HHVM) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/apache2-hhvm.conf`.
- * Rely on Nginx (PHP and HHVM) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/nginx.conf`.
-
-I would recommend you to rely on Apache2 as Nginx does not support TRACE request. When, your web server is configured
-and running, you will need to increase the number of fastcgi child processes (otherwise the main process will crash)
-and then, start it with the following commands:
+copy/paste the `phpunit.xml.dist` to `phpunit.xml` and uncomment the `TEST_SERVER` php server variable. Then, just 
```

Comment:
Oh, so much happened, I haven’t looked at the new repos yet - but it looks promising, thanks for the hint!

On 29 Apr 2015, at 0:55, Márk Sági-Kazár wrote:

>> @@ -48,21 +48,11 @@ $ bin/phpunit
>> ```
>>
>> This command will only run tests which does not require a real web server. To run the full tests, you will need to
>> -copy/paste the `phpunit.xml.dist` to `phpunit.xml` and uncomment the `TEST_SERVER` php server variable. Then, you have
>> -three possibilities:
>> -
>> - * Rely on Apache2 (PHP) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/apache2-php.conf`.
>> - * Rely on Apache2 (HHVM) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/apache2-hhvm.conf`.
>> - * Rely on Nginx (PHP and HHVM) by using the `tests/Ivory/Tests/HttpAdapter/Fixtures/etc/nginx.conf`.
>> -
>> -I would recommend you to rely on Apache2 as Nginx does not support TRACE request. When, your web server is configured
>> -and running, you will need to increase the number of fastcgi child processes (otherwise the main process will crash)
>> -and then, start it with the following commands:
>> +copy/paste the `phpunit.xml.dist` to `phpunit.xml` and uncomment the `TEST_SERVER` php server variable. Then, just
>
> It is already done in the new org repos. @jeromegamez I actually think you could improve yours based on that. ;)
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/egeloen/ivory-http-adapter/pull/107/files#r29297039